webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/Breadcrumb.js

```diff
@@ -3,8 +3,16 @@ import PropTypes from 'prop-types';
 import { Breadcrumb as PfBreadcrumb } from 'patternfly-react';
 import 'patternfly-react/dist/sass/_breadcrumb.scss';
 
+function truncate(text, length) {
+  if (length == null || text.length <= length) {
+    return text;
+  }
+  const truncated = text.slice(0, length);
+  return `${truncated} ...`;
```

Comment:
just wondering, can't we reuse ellipsis instead?

Reply:
Do you mean `text-overflow: ellipsis` css property or some other thing?

Reply:
I do and we do have a component for it, @amirfefer can you have a look please ?

Reply:
I tried wrapping the text in `EllipsisWithTooltip` but the result wasn't exactly what I expected. Guess I'd have to somehow confine the size of the entire bar.

![with ellipsis](https://i.imgur.com/gO9BLBs.png)